Update naming and images for stack trace and event navigator (#9043)
getsentry/sentry-docs: docs/product/issues/issue-details/index.mdx

![Issue details](issue_details_page.png)

> While the <SandboxLink scenario="oneIssue" projectSlug="react">Issue Details</SandboxLink> page displays information about a specific event that's part of an issue. You can navigate between the events of an issue using the event navigator.

Reviewer: ```suggestion
You can view information about a specific error event that's part of an issue on the <SandboxLink scenario="oneIssue" projectSlug="react">Issue Details</SandboxLink> page. You can also navigate between the events of an issue using the event navigator.
```

In the right hand sidebar, [sentry.io](https://sentry.io) reflects a summary that includes information such as how often the <SandboxLink scenario="oneIssue" projectSlug="react">issue</SandboxLink> has occurred in the last 24 hours and the last 30 days, as well as the last time and the first time the issue was seen. If the issue is linked to any GitHub or Jira issues, that's displayed here as well. Lastly, there is a facet map, which shows the distribution of values for the tags for all events included in the issue. All of these values are based on the environment that you've selected in the environment dropdown.

## Issue Categories & Page Details

There are two categories of issues: [error issues](/product/issues/issue-details/error-issues/) and [performance issues](/product/issues/issue-details/performance-issues/). Depending on the issue category and available event data, the **Issue Details** page displays some combination of the following sections:

> - **Event Navigator** - Allows you to select the latest, oldest, and recommended event from an issue and navigate between events in sequence.

Reviewer: ```suggestion
- **Event Navigator** - Allows you to select between the latest, oldest, or a recommended issue event and navigate between events in sequence.
```
